### DISCUSSION TITLE
Force auto skip on maps with very long lead in time

### DISCUSSION THREAD

**Discussion by HonKLam** [ppy/osu#27084] [Ideas]
### Type

Game behaviour

### Bug description

Normally, when you play a map with a long intro (or in my case a part of a marathon map), you are able to skip it in order to play immediately.

This functionality is working in Singleplayer mode, but not in Multiplayer mode.

How to reproduce:
- Open Multiplayer lobby
- Select part of a marathon map / map with long intro (tested with [this map](https://osu.ppy.sh/beatmapsets/1329091#osu/2759774))
- Start lobby
-> Can't skip first 40 minutes of map to play -> you are stuck waiting

### Screenshots or videos

https://github.com/ppy/osu/assets/62069795/ea6f2658-87de-4128-9c92-b90900804991

### Version

2024.131.0-lazer

### Logs

[compressed-logs.zip](https://github.com/ppy/osu/files/14201813/compressed-logs.zip)

**Comment by bdach** [comment]
No skip is generally intentional. Otherwise some players could be ahead of the rest of the room. By some minutes too in this case.

There is an auto skip setting that the room host can enable but it will take effect for everyone in the room.

I'm not sure why someone would pick a marathon map for multi, but maybe there's an argument that in such a case auto skip should engage regardless of the setting? That's more of a feature request though so I'm going to move this to discussions.

**Comment by HonKLam** [comment]
I wasn't aware of the auto skip setting in multi, thanks!

Either way it would be quite inconvenient if the setting is off and other players in multi are not aware of it and in the rare case of someone putting in a part of a marathon map, it can't be played and the lobby has to be aborted.

Enforcing a skip at a specific lead in time sounds good, but where do you draw the line? 30min? 5 min? 1 min? Not quite sure about this one

**Comment by respektive** [comment]
I'm wondering, wouldn't a vote skip functionality, like it's present on stable multi lobbies, solve this problem? (https://github.com/ppy/osu/discussions/26273)

**Reply by HonKLam** [reply]
That's what I was thinking too. I just figured that there had a to be a reason why they decided to implement the force-skip setting in multi lobbies. But I wouldn't mind reimplementing the vote feature from stable as well.

**Reply by bdach** [reply]
Probably would.
